### PR TITLE
fix(binanceus) - ratelimits 2

### PR DESCRIPTION
**Pull request by ttodua**
gradually adding endpoints,
tomorrow i'll push the final pr.

### CODE REVIEW

File: ts/src/binanceus.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export default class binanceus extends binance {
                 'api': {
                     'web': 'https://www.binance.us',
                     'sapi': 'https://api.binance.us/sapi/v1',
+                    'sapi_v3': 'https://api.binance.us/sapi/v3',
```
Comment (carlosmiei) [inline ts/src/binanceus.ts:22]: Hum, is this a new category?

Comment (ttodua) [reply ts/src/binanceus.ts:22]: seems so, look at the different v1/v3 between these two:
https://docs.binance.us/#get-sub-account-assets
https://docs.binance.us/#get-master-account-39-s-total-usd-value

Comment (ttodua) [reply ts/src/binanceus.ts:22]: right, binance is not going to use these paths at this moment, however, I've just added with the intention that all endpoints were available in binanceus so users could implicitly call them.

Comment (ttodua) [reply ts/src/binanceus.ts:22]: at this moment, it is a bit unclear foir me too, how we should make binance.ts (which should be agnostic) to use those unknown paths. however, let's put that step for a bit future, and at this moment, implicit endpoints calls can be supported.

File: ts/src/binanceus.ts [hunk 2]
```diff
@@ -102,6 +103,28 @@ export default class binanceus extends binance {
...
+                    'get': {
+                        'apiTradingStatus': 1,
+                        'asset/assetDistributionHistory': 1,
+                        'asset/query/trading-fee': 1,
+                        'asset/query/trading-volume': 1,
+                        'sub-account​/spotSummary': 1,
+                        'sub-account​/status': 1,
+                    },
+                },
+                'sapiV3': {
```
Comment (carlosmiei) [inline ts/src/binanceus.ts:116]: hum actually we have already a sapiV3 in binance 
image

**Comment by ttodua** [comment]
closing this PR in favor of this - https://github.com/ccxt/ccxt/pull/20954

**Pull request closed without merging**